### PR TITLE
feat: write setup guide for firefox for android

### PR DESCRIPTION
**Pull request by pavanjoshi914**
### Describe the changes you have made in this PR

initial version of setup guide to run and debug extension on firefox android

### Type of change

(Remove other not matching type)

- `feat`: New feature (non-breaking change which adds functionality)

### Checklist

- [X] Self-review of changed code
- [X] Manual testing
- [X] Added automated tests where applicable
- [X] Update Docs & Guides
- For UI-related changes
- [X] Darkmode
- [X] Responsive layout

### CODE REVIEW

File: doc/SETUP.md [hunk 1]
```diff
@@ -42,6 +42,29 @@
...
+
+1. Install or update [web-ext](https://github.com/mozilla/web-ext) to version 4.1.0 or later.
+2. Install the [Android Platform Tools](https://developer.android.com/tools/releases/platform-tools) (you can use Android studio's [sdk manager](https://developer.android.com/studio/intro/update.html#sdk-manager) or [sdk manager](https://developer.android.com/tools/sdkmanager) command line tools to install android platform tools)
+3. Make sure you have adb installed and in your PATH.
+
+b. **Setup Android Emulator**
+
+1.  Install [Firefox for Android Nightly](https://play.google.com/store/apps/details?id=org.mozilla.fenix) on Android
+2.  Enable [Android USB debugging](https://developer.android.com/studio/debug/dev-options) on the device.
+3.  Attach your device to the development computer using a USB cable. When prompted, allow USB debugging for the connection.
```
Comment (rolznz) [inline doc/SETUP.md:57]: step b above says use an emulator but these instructions above are for a real device, right?

File: doc/SETUP.md [hunk 2]
```diff
@@ -42,6 +42,29 @@
...
+4.  In the settings view for Firefox for Android Nightly, enable "Remote debugging via USB."
+5.  Run `$ adb devices` in the command shell to get the device code
+
+c. **Install and run extension**
+
+1. Remove native messaging permission from extensions manifest (for development environment specifically, this needs to be done)
```
Comment (rolznz) [inline doc/SETUP.md:63]: maybe we should note this is a temporary thing, right? until we find a proper solution?

File: doc/SETUP.md (hunk 1 above)
Comment (pavanjoshi914) [reply doc/SETUP.md:57]: changed that to device

File: doc/SETUP.md [hunk 3]
```diff
@@ -42,6 +42,29 @@
 
 To connect to a remote development LND node you can use a [test account](https://github.com/bumi/lightning-browser-extension/wiki/Test-setup)
 
+## Install and debug extension in Firefox Android
```
[the remark this answers is not in the document]
Comment (pavanjoshi914) [reply doc/SETUP.md:45]: resolved

File: doc/SETUP.md (hunk 2 above)
Comment (pavanjoshi914) [reply doc/SETUP.md:63]: resolved

**Review by rolznz — APPROVED** [review]
utACK, documentation looks great. Thanks @pavanjoshi914 !

File: doc/SETUP_ANDROID.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+2. Install the [Android Platform Tools](https://developer.android.com/tools/releases/platform-tools) (you can use Android studio's [sdk manager](https://developer.android.com/studio/intro/update.html#sdk-manager) or [sdk manager](https://developer.android.com/tools/sdkmanager) command line tools to install android platform tools)
+3. Make sure you have adb installed and in your PATH.
+
+b. **Setup Android Device**
+
+1.  Install [Firefox for Android Nightly](https://play.google.com/store/apps/details?id=org.mozilla.fenix) on Android
+2.  Enable [Android USB debugging](https://developer.android.com/studio/debug/dev-options) on the device.
+3.  Attach your device to the development computer using a USB cable. When prompted, allow USB debugging for the connection.
+4.  In the settings view for Firefox for Android Nightly, enable "Remote debugging via USB."
+5.  Run `$ adb devices` in the command shell to get the device code
```
Comment (reneaaron) [inline doc/SETUP_ANDROID.md:15]:
```suggestion
5.  Run `$ adb devices` in the command shell to get the device ID
```

**Review by reneaaron — APPROVED** [review]

**Pull request merged**